apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -118,72 +176,180 @@ export default class ExportHTMLModal extends Vue {
+    ),
+    // Medium severity; orange
+    circleMedium: this.iconDatatoSVG(
+      mdiCircle,
+      'rgb(255, 152, 0)',
+      'An orange circle'
+    ),
+    // High severity; deep orange
+    circleHigh: this.iconDatatoSVG(
+      mdiCircle,
+      'rgb(255, 87, 34)',
+      'A deep orange circle'
+    ),
+    // Critical severity; red
+    circleCritical: this.iconDatatoSVG(
+      mdiCircle,
+      'rgb(244, 67, 54)',
+      'A red circle'
+    ),
+    // Generic circle
+    circleWhite: this.iconDatatoSVG(mdiCircle, 'rgb(0, 0, 0)', 'A white circle')
+  };
+
+  // Default attributes
   showingModal = false;
   exportType = 'executive';
-  description = 'Profile Info\nStatuses\nCompliance Donuts';
+  description = 'Profile Info\nStatuses\nCompliance Level';
+  printHelp = false;
   outputData: OutputData = {
-    bootstrapCSS: '',
-    bootstrapJS: '',
-    jquery: '',
+    twStyles: '',
+    twElements: '',
+    // Used for profile status reporting
     statistics: {
       passed: 0,
       failed: 0,
       notApplicable: 0,
       notReviewed: 0,
+      profileError: 0,
+      totalControls: 0,
       passedTests: 0,
       passingTestsFailedControl: 0,
       failedTests: 0,
       totalTests: 0
     },
+    // Used for profile severity reporting
+    severity: {
+      none: 0,
+      low: 0,
+      medium: 0,
+      high: 0,
+      critical: 0
+    },
+    // Used for profile compliance reporting
+    compliance: {
+      level: '',
+      color: ''
+    },
     files: [],
     controlSets: [],
     showControlSets: false,
     showCode: false,
     exportType: '',
-    complianceCards: [],
+    // Series of icons used for profile-related detail reports
     icons: {
-      circleCheck: this.iconDatatoSVG(mdiCheckCircle, 'white'),
-      circleCross: this.iconDatatoSVG(mdiCloseCircle, 'white'),
-      circleMinus: this.iconDatatoSVG(mdiMinusCircle, 'white'),
-      circleAlert: this.iconDatatoSVG(mdiAlertCircle, 'white')
+      // Passed
+      circleCheck: this.iconHTMLStore.circleCheck,
+      // Failed
+      circleCross: this.iconHTMLStore.circleCross,
+      // Not applicable
+      circleMinus: this.iconHTMLStore.circleMinus,
+      // Not reviewed
+      circleAlert: this.iconHTMLStore.circleAlert,
+      // Profile error
+      triangleAlert: this.iconHTMLStore.triangleAlert,
+      // Total count
+      squareEqual: this.iconHTMLStore.squareEqual,
+      // No severity
+      circleNone: this.iconHTMLStore.circleNone,
+      // Low severity
+      circleLow: this.iconHTMLStore.circleLow,
+      // Medium severity
+      circleMedium: this.iconHTMLStore.circleMedium,
+      // High severity
+      circleHigh: this.iconHTMLStore.circleHigh,
+      // Critical severity
+      circleCritical: this.iconHTMLStore.circleCritical
     }
   };
 
+  // Configures outputData object's report type based on user input
   @Watch('exportType')
   onFileChanged(newValue: string, _oldValue: string) {
     switch (newValue) {
```

Comment:
there should always be a default on a switch statement if we're not doing it on something with predefined values like an enum - here we have something that's stringly-typed but at the end of the day is still a string so anything could be substituted in.  if possible, i would actually like to replace this with a straight up actual enum if at all possible.

Reply:
Changed as requested.